api/app/serializers/spree/api/v2/platform/payment_method_serializer.rb

```diff
@@ -3,7 +3,13 @@ module Api
     module V2
       module Platform
         class PaymentMethodSerializer < BaseSerializer
-          include ResourceSerializerConcern
+          attributes :name, :type, :description, :active, :display_on, :auto_capture, :position, :created_at, :updated_at, :deleted_at
```

Comment:
@MatthewKennedy why not use `ResourceSerializerConcern` here?

Reply:
This is one of those that pulls through all the strange attributes from the display on frontend backend both, the same as the shipping methods issue.

Reply:
Ah, that one, ok!